pkg/linter/linter_test.go

```diff
@@ -292,20 +293,91 @@ func TestLinters(t *testing.T) {
 		dirFunc: mkfile(t, "usr/lib/libcuda.so.560.35.05"),
 		linter:  "cudaruntimelib",
 		pass:    false,
+	}, {
+		dirFunc: mkfile(t, "usr/share/man/man1/foo.1"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    false,
+	}, {
+		dirFunc: mkfile(t, "usr/share/man/man1/foo.1"),
+		linter:  "maninfo",
+		pkg:     "regular-doc",
+		pass:    true,
+	}, {
+		dirFunc: mkfile(t, "usr/share/man/man8/bar.8.gz"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    false,
+	}, {
+		dirFunc: mkfile(t, "usr/local/share/man/man3/baz.3"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    false,
+	}, {
+		dirFunc: mkfile(t, "usr/man/man5/qux.5"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    false,
+	}, {
+		dirFunc: mkfile(t, "usr/share/info/test.info"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    false,
+	}, {
+		dirFunc: mkfile(t, "usr/share/info/test.info.gz"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    false,
+	}, {
+		dirFunc: mkfile(t, "usr/share/info/test.info-1"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    false,
+	}, {
+		dirFunc: mkfile(t, "usr/share/info/dir"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    false,
+	}, {
+		dirFunc: mkfile(t, "usr/share/info/test.info"),
+		linter:  "maninfo",
+		pkg:     "regular-doc",
+		pass:    true,
+	}, {
+		dirFunc: mkfile(t, "usr/bin/normal"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    true,
+	}, {
+		dirFunc: mkfile(t, "usr/local/share/info/test.info"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    false,
+	}, {
+		dirFunc: mkfile(t, "usr/lib/libfoo.so.1"),
+		linter:  "maninfo",
+		pkg:     "regular-package",
+		pass:    true,
 	}} {
 		ctx := slogtest.Context(t)
 		t.Run(c.linter, func(t *testing.T) {
 			dir := c.dirFunc()
+
+			packageName := c.pkg
+			if packageName == "" {
+				packageName = c.linter // Fallback for existing tests
```

Comment:
"for existing tests" helps a reviewer read this change, but it won't make a lot of sense to the majority of people in the future who read this and don't know what order things happened in. I'd suggest just saying something like "fallback to using the linter name as the fake package name"